pkg/storer/internal/cache/cache_test.go

```diff
@@ -295,63 +249,13 @@ func TestCache(t *testing.T) {
 			}
 		})
 	})
-	t.Run("handle error", func(t *testing.T) {
```

Comment:
It would be good to keep error checking tests.

Reply:
done